std/range/package.d

```diff
@@ -10899,7 +10899,20 @@ cost $(BIGOH n), use $(REF isSorted, std,algorithm,sorting).
 auto assumeSorted(alias pred = "a < b", R)(R r)
 if (isInputRange!(Unqual!R))
 {
-    return SortedRange!(Unqual!R, pred)(r);
+    // Avoid senseless `SortedRange!(SortedRange!(...), pred)` nesting.
+    static if (is(R == SortedRange!(RRange, RPred, Etc), RRange, alias RPred, Etc...))
```

Comment:
Why `Etc`?
AFAICT SortedRange only has `Range` + `pred`:

https://github.com/dlang/phobos/blob/f8a17a7898fd2a995f2f3fb1aac58f00f1ef827b/std/range/package.d#L10278-L10279

Reply:
I didn't want to have to update this if that template ever gained additional parameters. It may have made more sense when this is was `SortedRange!(RRange, Etc)`.

Reply:
Removed the unnecessary `Etc`.